portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +17,3 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
```

Comment:
You can change what the JS does, but you should leave *some* functionality since writing interactive pages is part of the experience

Reply:
I added JS functionality

Reply:
Resolving because I add JS interactive functionality